src/controllers/reset.controller.js

```diff
@@ -0,0 +1,76 @@
+import jwt from 'jsonwebtoken';
+import { config } from 'dotenv';
+import ResetService from '../services/reset.service';
+import UserAuth from '../services/send-email.service';
+import response from '../helpers/response';
+import authManager from '../helpers/jwt';
+import BcryptService from '../services/bcrypt.service';
+
+config();
+
+// const response = new Response();
+
+/**
+* class user controller
+*/
+class ResetController {
+  /**
+  * @param {Request} req
+  * @param {Response} res
+  * @returns {FindUser} the function find if user exist in database
+  */
+  static async findUser(req, res) {
+    const { email } = req.body;
+    const user = await ResetService.findUser(email);
+
+    if (!user) {
```

Comment:
@erickyvand , you need to move all validations into middleware file. only controller must handle valid data